src/components/Feedback.vue

```
@@ -73,14 +73,36 @@
+  alert(JSON.stringify(session))
+
+  const opts = {
+    host: apiUrl,
+    path,
+    method,
+    body: JSON.stringify({ name, email, message }),
+    headers: { 'Content-Type': 'application/json' }
+  }
+
+  aws4.sign(opts, {
+    accessKeyId: credentials.accessKeyId,
```

ESLint

[error] 'credentials' is not defined.

(no-undef)